src/components/StrongPasswordInput/index.js

```diff
@@ -0,0 +1,203 @@
+import { useUniqueIdentifier, useErrorMessageId, useLabelId } from '../../libs/hooks';
+import usePasswordState from './hooks/usePasswordState';
+
+/**
+ * StrongPasswordInput component is used to provide feedback about password strength
+ * @category Form
+ */
+const StrongPasswordInput = React.forwardRef((props, ref) => {
+    const {
+        id,
+        className,
+        style,
+        bottomHelpText,
+        error,
+        passwordState,
+        passwordStateLabel,
+        value,
+        placeholder,
+        onChange,
+        disabled,
+        readOnly,
+        tabIndex,
+        onFocus,
+        onBlur,
+        onClick,
+        onKeyDown,
+        maxLength,
+        minLength,
+        icon,
+        required,
+        name,
+        label,
+        hideLabel,
+        iconPosition,
+    } = props;
+
+    const inputId = useUniqueIdentifier('input');
+    const errorMessageId = useErrorMessageId(error);
+    const labelId = useLabelId(label);
+    const status = usePasswordState(passwordState);
+
+    return (
+        <StyledContainer id={id} className={className} style={style}>
+            <Label
+                label={label}
+                hideLabel={hideLabel}
+                required={required}
+                inputId={inputId}
+                readOnly={readOnly}
+                id={labelId}
+            />
+
+            <RelativeElement>
+                <RenderIf isTrue={!!icon}>
+                    <StyledIconContainer
+                        iconPosition={iconPosition}
+                        readOnly={readOnly}
+                        error={error}
+                    >
+                        {icon}
+                    </StyledIconContainer>
+                </RenderIf>
+
+                <StyledInput
+                    id={inputId}
+                    name={name}
+                    type="password"
+                    value={value}
+                    placeholder={placeholder}
+                    onChange={onChange}
+                    tabIndex={tabIndex}
+                    onFocus={onFocus}
+                    onBlur={onBlur}
+                    onClick={onClick}
+                    onKeyDown={onKeyDown}
+                    disabled={disabled}
+                    readOnly={readOnly}
+                    required={required}
+                    maxLength={maxLength}
+                    minLength={minLength}
+                    aria-labelledby={labelId}
+                    aria-describedby={errorMessageId}
+                    iconPosition={iconPosition}
+                    icon={icon}
+                    error={error}
+                    status={status}
+                    ref={ref}
+                />
+            </RelativeElement>
+            <StrengthBar passwordState={passwordState} passwordStateLabel={passwordStateLabel} />
+            <RenderIf isTrue={!!bottomHelpText}>
+                <HelpText alignSelf="center" style={{ margin: 0 }}>
```

Comment:
no inline styles let's maintain the separation of concerns principal, use a styled instead